IndomitableIbises/Bangazon-API-Sprint-1
# Cashew and I got the nested products to work on Orders

Link to Ticket
Closes #10 

Added a many to many table between order and product.  It is called Prod_Order.  When looking at an order it now shows each Product within that order.    

Steps to Test
Outline the steps to test

git fetch --all
git checkout JCproductorder

Make sure your server isn't running
In Bash/Terminal, run ./django_data.sh API faker_factory for PC or django_data.sh API faker_factory for Mac
Run server: python manage.py runserver
Click on orders link.  Make sure under products it is listing individual products
Ask me questions should you have any
Impacted Areas in Application
List general components of the application that this PR will affect:

*views.py
*urls.py
*serializer.py

Mentions @username
@CashewRose @rafcevallos  @sirwin4

Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each
